gulpfile.babel.js

```diff
@@ -220,5 +229,44 @@ export const build = gulp.series(
 	gulp.parallel(processHtml, lint, images, clientScripts, vendorScripts, styles, bundleExternalCSS, copyStaticFiles, lintBootstrap, security)
 );
 
+function bumpVersion() {
+	let type;
+	if (args.major) {
+		type = 'major';
+	} else if (args.minor) {
+		type = 'minor';
+	} else if (args.patch) {
+		type = 'patch';
+	} else {
+		onError('Please specify release type --(major|minor|patch)');
+	}
+	return gulp.src('./package.json')
+		.pipe(bump({type}))
+		.on('error', onError)
+		.pipe(gulp.dest('.'));
+}
+
+function createChangelog() {
+	return gulp.src('./CHANGELOG.md', {buffer: false})
+		.pipe(changelog({preset: 'angular'}))
+		.pipe(gulp.dest('./'));
+}
+
+function commitChanges() {
+	return gulp.src('.')
+		.pipe(git.add())
+		.pipe(git.commit('[Pre-Release] Bumped version number'));
```

Comment:
Or "Release v4.0.0". That's what we used to generate before. But I don't care. It's up to you.